src/components/menu/ComingSoon/ComingSoon.js

```diff
@@ -0,0 +1,44 @@
+import './GoogleBrandFormatting.css'
+
+
+function ComingSoon ()
+{
+    return ( 
+        <div className='center'>  
+<h1 className='h1-green'> <span class = "h1-blue">
+C
+    </span>
+    <span class = "h1-red">
+o
+    </span>
+    <span class = "h1-yellow">
+m
+    </span>
+    <span class = "h1-blue">
+i
+    </span>
+    <span class = "h1-green">
+n
+    </span>
+g  <span class = "h1-red">
+    </span>
+    <span class = "h1-blue">
+S
+    </span>
+    <span class = "h1-red">
+o
+    </span>
+    <span class = "h1-yellow">
+o
+    </span>
+n... </h1>   
+        <br/>  
+       <h2 className='h2-white'> 
+       This page is currently under development and we hope it will be completed soon!
+       </h2>   
+       <p className='p-green'> If you're interested in seeing our progress you can check our GitHub.</p> 
```

Comment:
I like how you added a reference to our GitHub, let's make this a clickable link

Reply:
Love the idea! Completed that!